api/src/main/java/org/pmiops/workbench/notebooks/NotebooksServiceImpl.java

```diff
@@ -169,6 +174,20 @@ public void saveNotebook(String bucketName, String notebookName, JSONObject note
         notebookContents.toString().getBytes(StandardCharsets.UTF_8));
   }
 
+  @Override
+  public String getReadOnlyHtml(
+      String workspaceNamespace, String workspaceName, String notebookName) {
+    String bucketName =
+        fireCloudService
+            .getWorkspace(workspaceNamespace, workspaceName)
+            .getWorkspace()
+            .getBucketName();
+    String notebook =
+        getNotebookContents(bucketName, notebookName).toString(); // this might be wrong
+
+    return fireCloudService.staticNotebooksConvert(notebook);
```

Comment:
This could be a follow-up, but we should have a sanitization step here. At a glance, this looks pretty straightforward to use: https://github.com/OWASP/java-html-sanitizer

nbconvert does do sanitization, but we should practice defense in depth